src/cpp/benders/benders_core/WorkerMaster.cpp

```diff
@@ -229,59 +253,65 @@ WorkerMaster::WorkerMaster(Str2Int const & variable_map, std::string const & pat
 	_solver->get_ub(bounds.data(), 0, ncols - 1);
 	CharVector bndTypes(ncols, 'U');
 	IntVector indices(ncols);
-	for (int i = 0; i < _solver->get_ncols(); i++) {
+	for (int i = 0; i < _solver->get_ncols(); i++)
+	{
 		indices[i] = i;
 		bounds[i] = std::min(bounds[i], 1e20);
 	}
 	_solver->chg_bounds(indices, bndTypes, bounds);
 
 	// add the variable alpha
 	auto const it(_name_to_id.find("alpha"));
-	if (it == _name_to_id.end()) {
+	if (it == _name_to_id.end())
+	{
 		double lb(-1e10); /*!< Lower Bound */
 		double ub(+1e20); /*!< Upper Bound*/
 		double obj(+1);
 		std::vector<int> start(2, 0);
 		_id_alpha = _solver->get_ncols(); /* Set the number of columns in _id_alpha */
 
-        solver_addcols(_solver, DblVector(1, obj), IntVector(1, 0), IntVector(0, 0), DblVector(0, 0.0),
-                       DblVector(1, lb), DblVector(1, ub), CharVector(1, 'C'), StrVector(1, "alpha")); /* Add variable alpha and its parameters */
+		solver_addcols(_solver, DblVector(1, obj), IntVector(1, 0), IntVector(0, 0), DblVector(0, 0.0),
+					   DblVector(1, lb), DblVector(1, ub), CharVector(1, 'C'), StrVector(1, "alpha")); /* Add variable alpha and its parameters */
 
 		_id_alpha_i.resize(nslaves, -1);
-		for (int i(0); i < nslaves; ++i) {
+		for (int i(0); i < nslaves; ++i)
+		{
 			std::stringstream buffer;
 			buffer << "alpha_" << i;
 			_id_alpha_i[i] = _solver->get_ncols();
-            solver_addcols(_solver, DblVector(1, 0.0), IntVector(1, 0), IntVector(0, 0), DblVector(0, 0.0),
-                           DblVector(1, lb), DblVector(1, ub), CharVector(1, 'C'),
-                           StrVector(1, buffer.str())); /* Add variable alpha_i and its parameters */
+			solver_addcols(_solver, DblVector(1, 0.0), IntVector(1, 0), IntVector(0, 0), DblVector(0, 0.0),
+						   DblVector(1, lb), DblVector(1, ub), CharVector(1, 'C'),
+						   StrVector(1, buffer.str())); /* Add variable alpha_i and its parameters */
```

Comment:
There seems to be a lot of optimization-related stuff here. Maybe move it somewhere else ? At least create methods/functions such as `add_alpha`.